content/blog/2024-02-25-announcing-poetry-1-8-0.md

```diff
@@ -66,7 +66,7 @@ has to download the whole file to calculate the hash itself.
 Although we tested this feature with various package indices,
 there still might be servers that respond to HTTP range requests in an unexpected way.
 Therefore, if you experience any issues with this feature,
-you can [disable lazy-wheel](docs/master/configuration#solverlazy-wheel).
+you can [disable lazy-wheel](docs/main/configuration#solverlazy-wheel).
```

Comment:
This link is a 404 in the preview. Is this just a preview issue?

Reply:
Something is off, looking.

Reply:
It is also 404 on live version the rel link is wrong I think.

```
https://python-poetry.org/blog/announcing-poetry-1.8.0/docs/master/configuration#solverlazy-wheel
```